src/towncrier/_project.py

```diff
@@ -63,56 +62,50 @@ def get_version(package_dir: str, package: str) -> str:
     Try to extract the version from the distribution version metadata that matches
     `package`, then fall back to looking for the package in `package_dir`.
     """
-    version: Any
+    version: str
 
     # First try to get the version from the package metadata.
     if version := _get_metadata_version(package):
         return version
 
     # When no version if found, fall back to looking for the package in `package_dir`.
     module = _get_package(package_dir, package)
-
     version = getattr(module, "__version__", None)
-
     if not version:
-        raise Exception(
-            f"No __version__ or metadata version info for the '{package}' package."
-        )
+        try:
```

Comment:
I this change needed ?

I was thinking that this was sorted out in https://github.com/twisted/towncrier/pull/502

Reply:
Right, I think it's just a merge artifact. Reverted